docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -207,6 +207,22 @@ This means that the number of rows will drive the height of the grid and consequ
 
 {{"demo": "pages/components/data-grid/rendering/AutoHeightGrid.js", "bg": "inline"}}
 
+## Size
```

Comment:
How about we use the two wordings at the same time so developers, when searching, e.g. with Algolia can find the content?

```suggestion
## Size/Density
```

Reply:
done